weather/weather.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+import pyowm
+import os
+
+API_key = "229c25e22cfd9c434134d5c88a1ae588"
```

Comment:
```suggestion
API_key = "" # insert your API key
if not key:
    print("No API key specified")
```
Please remove any secrets from your code.